src/template.txt

```diff
@@ -417,18 +429,18 @@ define(<%= imports %>, function(<%= args %>){
 
 		var viewModel = new ViewModel(params);
 
-		if(!canReflect.getKeyValue(viewModel, "statusCode") &&
-			!isEmptyObject(route.routes)) {
+		if(!canReflect.getKeyValue(viewModel, "statusCode") && !isEmptyObject(route.routes)) {
 			if(hasMatchingRoute) {
 				canReflect.setKeyValue(viewModel, "statusCode", 200);
 			} else {
 				canReflect.setKeyValue(viewModel, "statusCode", 404);
 				canReflect.setKeyValue(viewModel, "statusMessage", "Not found");
 			}
 		}
-
-		route.data = viewModel;
-		route.start();
+		if(!isEmptyObject(route.routes)) {
+			route.data = viewModel;
+			route.start();
+		}
```

Comment:
Does anything bad happen if we do this when there are no routes defined?

Reply:
@pYr0x Would like to get this in. Can we remove this if?

Reply:
i will remove it..